JitPack's `publishToMavenLocal` stopped working after upgrading to Gradle 8.x

Before: https://jitpack.io/com/github/elimu-ai/analytics/3.1.21/build.log
```
Tasks: publishToMavenLocal,
Running: ./gradlew clean -Pgroup=com.github.elimu-ai -Pversion=3.1.21 -xtest -xlint assemble publishToMavenLocal

> Configure project :utils
WARNING:Software Components will not be created automatically for Maven publishing from Android Gradle Plugin 8.0. To opt-in to the future behavior, set the Gradle property android.disableAutomaticComponentCreation=true in the `gradle.properties` file or use the new publishing DSL.
WARNING:We recommend using a newer Android Gradle plugin to use compileSdk = 33
.
.
.
✅ Build artifacts:
com.github.elimu-ai:analytics:3.1.21

Files: 
com/github/elimu-ai/analytics/3.1.21
com/github/elimu-ai/analytics/3.1.21/analytics-3.1.21-sources.jar
com/github/elimu-ai/analytics/3.1.21/analytics-3.1.21.aar
com/github/elimu-ai/analytics/3.1.21/analytics-3.1.21.pom
com/github/elimu-ai/analytics/3.1.21/analytics-3.1.21.pom.md5
com/github/elimu-ai/analytics/3.1.21/analytics-3.1.21.pom.sha1
com/github/elimu-ai/analytics/3.1.21/build.log
```

---

After: https://jitpack.io/com/github/elimu-ai/analytics/3.1.22/build.log
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':utils:publishUtilsPublicationToMavenLocal' (type 'PublishToMavenLocal').
  - Gradle detected a problem with the following location: '/home/jitpack/build/utils/build/outputs/aar/utils-3.1.22-release.aar'.
    
    Reason: Task ':utils:publishUtilsPublicationToMavenLocal' uses this output of task ':utils:bundleReleaseAar' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':utils:bundleReleaseAar' as an input of ':utils:publishUtilsPublicationToMavenLocal'.
      2. Declare an explicit dependency on ':utils:bundleReleaseAar' from ':utils:publishUtilsPublicationToMavenLocal' using Task#dependsOn.
      3. Declare an explicit dependency on ':utils:bundleReleaseAar' from ':utils:publishUtilsPublicationToMavenLocal' using Task#mustRunAfter.
    
    For more information, please refer to https://docs.gradle.org/8.6/userguide/validation_problems.html#implicit_dependency in the Gradle documentation.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 2m 51s
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
Publication: ai.elimu.analytics:utils:3.1.22
Build tool exit code: 1
2025-02-24T07:15:43.036049822Z
Exit code: 1

⚠️ ERROR: No build artifacts found
```

--- a/utils/build.gradle
+++ b/utils/build.gradle
@@ -34,7 +34,7 @@ dependencies {
 // Usage: ./gradlew clean build publish -PmavenUsername=***** -PmavenPassword=*****
 publishing {
     publications {
-        utils(MavenPublication) {
+        release(MavenPublication) {
             groupId 'ai.elimu.analytics'
             artifactId 'utils'
             version '3.1.22-SNAPSHOT'

